app/src/test/java/com/willowtree/vocable/screens/PresetsViewModelTest.kt

```diff
@@ -0,0 +1,131 @@
+package com.willowtree.vocable.screens
+
+import androidx.lifecycle.Observer
+import com.BaseUnitTest
+import com.nhaarman.mockitokotlin2.argumentCaptor
+import com.nhaarman.mockitokotlin2.capture
+import com.nhaarman.mockitokotlin2.whenever
+import com.willowtree.vocable.getOrAwaitValue
+import com.willowtree.vocable.mock
+import com.willowtree.vocable.presets.PresetCategories
+import com.willowtree.vocable.presets.PresetsRepository
+import com.willowtree.vocable.presets.PresetsViewModel
+import com.willowtree.vocable.room.Category
+import com.willowtree.vocable.room.CategoryDao
+import com.willowtree.vocable.room.Phrase
+import com.willowtree.vocable.room.VocableDatabase
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.*
+import org.mockito.Mockito.*
+import org.mockito.junit.MockitoJUnitRunner
+
+@ExperimentalStdlibApi
+@ExperimentalCoroutinesApi
+@RunWith(MockitoJUnitRunner::class)
+class PresetsViewModelTest : BaseUnitTest() {
+
+    private lateinit var presetsViewModel: PresetsViewModel
+
+    @Mock
+    private lateinit var categoryObserver: Observer<Category>
+
+    @Mock
+    private lateinit var categoryListObserver: Observer<List<Category>>
+
+    @Mock
+    private lateinit var phraseListObserver: Observer<List<Phrase>>
+
+    @Mock
+    private lateinit var presetsRepository: PresetsRepository
```

Comment:
I think we may want to move these to an interface pattern eventually. So we can have the "real" implementation that we use in the app and a "fake" one we use for tests. That way we don't have to use Mockito to mock responses.